doc: samples: central_and_peripheral_hr: Clarify documentation. (#6582)
nrfconnect/sdk-nrf: samples/bluetooth/central_and_peripheral_hr/README.rst

   :header: heading
   :rows: nrf5340dk_nrf5340_cpuapp_and_cpuapp_ns, nrf52840dk_nrf52840, nrf52dk_nrf52832, nrf52833dk_nrf52833, nrf52833dk_nrf52820, nrf21540dk_nrf52840

> For testing just the Bluetooth® LE Central Role operation the sample requires one of the following setups:

Reviewer: ```suggestion
To test just the Bluetooth® LE Central Role operation, you need the following setup:
```

Author: Fixed in a bit different way to emphasize the bullet list is an alternative.

Reviewer: Right, I misunderstood this. I thought the alternatives were a smartphone or tablet AND another development kit. Now that you corrected me, I think the original suggestion was better than this.

Author: done

  * A smartphone or a tablet running a compatible application.

  * Another development kit running the :ref:`zephyr:peripheral_hr` sample.
  See the documentation for that sample for detailed instructions.

> To test the Relay mode operation you need one of the following setups:

Reviewer: ```suggestion
To test the Relay mode operation, you need the following setup:
```

Author: Fixed in a bit different way to emphasize the bullet list is an alternative.

Reviewer: Same with this, but have the comma there after "operation" ;)

Author: done

Testing
=======

> After programming the sample to your development kit, you can test it either by connecting to other development kits that are running the :ref:`zephyr:peripheral_hr` sample, or by using the `nRF Connect for Desktop`_ **Bluetooth Low Energy** application that emulates a HRS server.

Reviewer: ```suggestion
After programming the sample to your development kit, test it either by connecting to other development kits that are running the :ref:`zephyr:peripheral_hr` sample, or by using the `nRF Connect for Desktop`_ **Bluetooth Low Energy** application that emulates a HRS server.
```

Author: Fixed. Also change "a" to "an" before "HRS" - hope that's ok.
